tests/test_utils.py

```diff
@@ -625,3 +654,99 @@ def test_csv_latin_1_encoding(self):
         self.assertEqual(lookup_dict[4]['header 2'], 5)
         self.assertEqual(lookup_dict[4]['header 3'], 'foo')
         self.assertEqual(lookup_dict[1]['header 1'], 1)
+
+
+class CreateCoordinateTransformationTests(unittest.TestCase):
+    """Tests for natcap.invest.utils.create_coordinate_transformer."""
+
+    def test_latlon_to_latlon_transformer(self):
+        """Utils: test transformer for lat/lon to lat/lon."""
+        from natcap.invest import utils
+
+        # Willamette valley in lat/lon for reference
+        lon = -124.525
+        lat = 44.525
+
+        base_srs = osr.SpatialReference()
+        base_srs.ImportFromEPSG(4326)  # WSG84 EPSG
+
+        target_srs = osr.SpatialReference()
+        target_srs.ImportFromEPSG(4326)
+
+        transformer = utils.create_coordinate_transformer(base_srs, target_srs)
+        x, y, _ = transformer.TransformPoint(lon, lat)
+
+        expected_x = -124.525
+        expected_y = 44.525

```

Comment:
Looks like we could do without these vars and just use the already defined `lon` and `lat` as the expected data?